regtests/bin/matrix_ncep

```diff
@@ -63,15 +71,26 @@
     modmpi='hpc-impi/2019.6'
     modnetcdf='netcdf/4.7.4'
     metispath='/work/noaa/marine/ali.abdolali/Source/hpc-stack/parmetis-4.0.3'
+    modjasper='jasper/2.0.25'
+    modzlib='zlib/1.2.11'
+    modpng='png/1.6.35'
+    modhdf5='hdf5/1.10.6'
+    modbacio='bacio/2.4.1'
+    modg2='g2/3.4.1'
+    modw3nco='w3nco/2.4.1'
+    modesmf='esmf/8_1_0_beta_snapshot_47'
```

Comment:
Now that esmf/8_1_1  is available, we could use that?  We can also update that later.

Reply:
I'll update it.

Reply:
I changed it and tested ww3_multi_esmf build with it.